### DISCUSSION TITLE
Mojo::Single

### DISCUSSION THREAD

**Discussion by ajmetz** [mojolicious/mojo#2217] [General]
Heard this, and thought of Perl Programming, =).
[https://www.youtube.com/watch?v=JvX19HX0yys](https://www.youtube.com/watch?v=JvX19HX0yys)

**Comment by guest20** [comment]
Might as well throw this in here: [Pearl Jam - Alive (Official Video)](https://www.youtube.com/watch?v=qM0zINtulhM)
